api/src/main/java/org/pmiops/workbench/interceptors/AuthInterceptor.java

```diff
@@ -171,7 +171,7 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
             new UserAuthentication(user, OAuth2Userinfo, token, UserType.RESEARCHER));
 
     // TODO: setup this in the context, get rid of log statement
```

Comment:
Might be worth converting this vague (and misleading?) TODO into an explainer comment describing why this log line is important. Channeling CH, it sounds like this is important to verify which user an authenticated request came from.